Assignment4/CountingIslands.java

```diff
@@ -0,0 +1,58 @@
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+ 
+public class CountingIslands
+{
+    int CountingIslands(int noRows, int noCols, boolean[][] landOrWater) 
+	{
+		int index, indexj;
```

Comment:
You've got a mixture of tabs and spaces, and it looks like your editor has tab == 4 spaces. When viewed on github, which thinks tab == 8 spaces, it looks wonky.

The best thing to do is be consistent, and always use just tabs or spaces. The Google coding style uses only spaces. You should be able to set your text editor up to always use just one or the other.